drivers/serial/uart_shell.c

```diff
@@ -102,14 +122,18 @@ static void device_name_get(size_t idx, struct shell_static_entry *entry)
 SHELL_DYNAMIC_CMD_CREATE(dsub_device_name, device_name_get);
 
 SHELL_STATIC_SUBCMD_SET_CREATE(sub_uart_cmds,
+	SHELL_CMD_ARG(write, &dsub_device_name,
+		      "Write data to the UART device\n"
+		      "Usage: write <device> <data>",
+		      cmd_uart_write, 3, 0),
 	SHELL_CMD_ARG(baudrate, &dsub_device_name,
-			  "Configure UART device baudrate\n"
-			  "Usage: baudrate <device> <baudrate>",
-			  cmd_uart_baudrate, 3, 0),
+		      "Configure UART device baudrate\n"
+		      "Usage: baudrate <device> <baudrate>",
+		      cmd_uart_baudrate, 3, 0),
 	SHELL_CMD_ARG(fc, &dsub_device_name,
-			  "Configure UART device flow control\n"
-			  "Usage: fc <device> <none|rtscts|dtrdsr|rs485>",
-			  cmd_uart_flow_control, 3, 0),
+		      "Configure UART device flow control\n"
+		      "Usage: fc <device> <none|rtscts|dtrdsr|rs485>",
+		      cmd_uart_flow_control, 3, 0),
```

Comment:
Please do not mix whitespace changes that are not related to your changes.

Reply:
I thought it would be too much to add a commit  just for indentation but ok, here you go.